README.md

```diff
@@ -6,6 +6,10 @@ BUILDING WITH VISUAL STUDIO
 ===========================
 Simply open `ZbarVS/ZbarVS.sln` using Visual Studio 2019 or above and build the project.
 
+BUILDING ON UBUNTU
+===========================
+Simply run build.sh to build the project.
+
```

Comment:
Please add description on modifying option in `build.sh` to build on other Linux distro.
Thanks.